===================================================================
RCS file: /cvs/gnome/gnome-vfs/libgnomevfs/gnome-vfs-volume-ops.c,v
retrieving revision 1.30
retrieving revision 1.30.2.1
diff -u -r1.30 -r1.30.2.1
--- gnome-vfs-volume-ops.c	2006/03/06 09:33:20	1.30
+++ libgnomevfs/gnome-vfs-volume-ops.c	2006/04/25 11:22:14	1.30.2.1
@@ -742,9 +742,9 @@
 
 			if (exit_status != 0) {
 				info->succeeded = FALSE;
-				if ((strcmp (info->argv[0], GNOME_VFS_BINDIR "/gnome-mount") == 0) ||
-				    (strcmp (info->argv[0], GNOME_VFS_BINDIR "/gnome-umount") == 0) ||
-				    (strcmp (info->argv[0], GNOME_VFS_BINDIR "/gnome-eject") == 0)) {
+				if ((strcmp (argv[0], GNOME_VFS_BINDIR "/gnome-mount") == 0) ||
+				    (strcmp (argv[0], GNOME_VFS_BINDIR "/gnome-umount") == 0) ||
+				    (strcmp (argv[0], GNOME_VFS_BINDIR "/gnome-eject") == 0)) {
 					/* gnome-mount programs display their own dialogs */
 					info->error_message = g_strdup ("");
 					info->detailed_error_message = g_strdup ("");
